src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -786,9 +786,11 @@ void SocketTraceConnector::TransferDataImpl(ConnectorContext* ctx) {
     } else {
       // If there's no transfer function, then the tracker should not be holding any data.
       // http::ProtocolTraits is used as a placeholder; the frames deque is expected to be
-      // std::monotstate.
-      ECHECK(conn_tracker->send_data().Empty<protocols::http::Message>());
-      ECHECK(conn_tracker->recv_data().Empty<protocols::http::Message>());
+      // std::monostate.
+      ECHECK((conn_tracker->send_data()
+                  .Empty<protocols::http::stream_id_t, protocols::http::Message>()));
+      ECHECK((conn_tracker->recv_data()
+                  .Empty<protocols::http::stream_id_t, protocols::http::Message>()));
```

Comment:
We should probably make this a `DCHECK`.

Also, is it possible to create a type alias so that the line breaks are avoided, e.g.
```
using stream_id_t = protocols::http::stream_id_t;
using message_t = protocols::http::Message;
.Empty<stream_id_t, message_t>();
```

Reply:
Added type alias and changed to `DCHECK`